lsp-mode.el

```diff
@@ -4277,8 +4279,13 @@ Also, additional data to attached to each candidate can be passed via PLIST."
                                  it))))
                   (-flatten-n 1)
                   (-sort (-on #'> (lambda (o)
-                                    (or (get-text-property 0 'completion-score o)
-                                        0))))
+                                    (or (get-text-property 0 'sort-score o)
```

Comment:
are we sure that this won't be persisted multiple completions when using cache?

Reply:
Yes, it won't be persisted. The list is created a new every time by previous block. The only thing persisted is `lsp-completion-item`.

Reply:
Wait, let me check again.

Reply:
Nvm, the list is indeed created a new every time by `completion-all-completions`.